src/api/org.ts

```diff
@@ -295,6 +299,9 @@ export function updateOrg(
         can_setup_saml: updateOrgRequest.canSetupSaml,
         metadata: updateOrgRequest.metadata,
         max_users: updateOrgRequest.maxUsers,
+        domain_autojoin: updateOrgRequest.canJoinOnEmailDomainMatch,
```

Comment:
I believe the backend expects this as `autojoin_by_domain` and `restrict_to_domain`